CHANGES.md

```diff
@@ -2,6 +2,19 @@
 
 ## __NEXT__
 
+### Features
+
+* Add the curate subcommand with two sub-subcommands, passthru and normalize-strings. The curate subcommand is intended to be a suite of commands to help users with data curation prior to running Nextstrain analyses. We will continue to add more subcommands as we identify other common data curation tasks. Please see the [usage docs](https://docs.nextstrain.org/projects/augur/en/stable/usage/cli/curate) for details. [#1039][] (@joverlee521)
+* Add pyfastx as a dependency to support random access of sequences in a FASTA file. [#1039][] (@joverlee521)
+* io: add functions for supporting the curate I/O framework. [#1039][] (@joverlee521)
+  * io.json: module to support JSONs and NDJSONs, originally copied from [seattleflu/id3c](https://github.com/seattleflu/id3c)
+  * io.read_table_to_dict(): read CSV and TSV files to yield each row as a dict
+  * io.read_metadata_with_sequences(): read CSV/TSV metadata files and a FASTA file to yield each row with a matching sequence as a dict
+  * io.write_records_to_tsv(): write each dict of an iterator as a row to an output TSV
+  * io.write_record_to_fasta(): write the sequence ID and sequence from each dict of an iterator to an output FASTA file
+* types: add `DataErrorMethod` enum to represent data error handling methods. [#1039][](@joverlee521)
```

Comment:
The expanded description of `curate` here and link to the new docs page is great!

I would omit the other entries you added though: they're all internal developer focused not end user focused, and a changelog for something like Augur is (IMO) best viewed/used as a document for end users.

(Sorry if I misled you here to include them with my earlier comment!)

Reply:
> I would omit the other entries you added though: they're all internal developer focused not end user focused, and a changelog for something like Augur is (IMO) best viewed/used as a document for end users.

That's fair, it's nice for external users if the changelog isn't bloated with internal dev changes!

Currently, we seem to be tracking both user and developer focused changes in the changelog. If we agree as a team to only add user focused entries to the changelog, then we should probably update the [PR template](https://github.com/nextstrain/augur/blob/master/.github/pull_request_template.md) to reflect this.

Reply:
Nod. When I looked earlier, the existing log seemed _mostly_ user-focused? with a light sprinkling of dev stuff… but I didn't skim too closely.

Agreed that updating PR template would be good.

Reply:
Removed the extra entries. 
We can continue discussion re: changelog in this [other PR](https://github.com/nextstrain/augur/pull/1089) 😄